test/widget/auth/auth_page_widget_test.dart

```diff
@@ -239,5 +239,107 @@ void main() {
       // check correct action is dispatched
       expect(testMiddleware.received(SignInWithGoogle()), true);
     });
+
```

Comment:
these tests were moved from the deleted "other_auth_options_page_widget_test.dart" as the UI from OtherAuthOptionsPage was moved into AuthPage - I'm pretty sure only some small renaming changes were made and the logic is all exactly the same.

Reply:
sweet!